src/Cache/_ScsDataClient.php

```diff
@@ -1,30 +1,34 @@
 <?php
 namespace Momento\Cache;
 
+use Cache_client\_DeleteRequest;
 use Cache_client\_GetRequest;
 use Cache_client\_SetRequest;
 use Cache_client\_SetResponse;
 
+use Momento\Cache\CacheOperationTypes\CacheDeleteResponse;
 use Momento\Cache\CacheOperationTypes\CacheGetResponse;
 use Momento\Cache\CacheOperationTypes\CacheSetResponse;
 use Momento\Utilities\_ErrorConverter;
 use function Momento\Utilities\validateTtl;
 use function Momento\Utilities\validateCacheName;
+use function Momento\Utilities\validateOperationTimeout;
 
 class _ScsDataClient
 {
 
-    private static int $DEFAULT_DEADLINE_SECONDS = 5;
+    private static int $DEFAULT_DEADLINE = 500000; // I believe this maps to 5 sec.
```

Comment:
so is this supposed to be in milliseconds now? Shouldnt it be 5000ms = 5 seconds? this is 50 seconds right now? Maybe missing something though
Also could we add unit to variable name as well so like  `$DEFAULT_DEADLINE_MS` if it is in ms.

Reply:
It definitely isn't in milliseconds, but I'm having a heck of a time figuring out what the unit **is** supposed to be. My closest rational guess is actually microseconds (!?). I've changed the code here to make this all a little more explicit and added a TODO comment for further research when time allows.

Reply:
yeah looks like µs looking again I think ur right